engine/runtime/src/test/java/org/enso/interpreter/test/DebuggingEnsoTest.java

```diff
@@ -18,10 +18,12 @@
 import org.junit.Assert;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class DebuggingEnsoTest {
   @Test
+  @Ignore
```

Comment:
Tracked by https://www.pivotaltracker.com/story/show/183798184

Reply:
OK to temporarily disable. However @jdunkerley & others are using `enso --inspect` option and right now it not going to show values of local variables - e.g. it's usefulness is going to be seriously impacted.

Reply:
Done.